oneview/resource_logical_enclosure.go

```diff
@@ -171,10 +177,12 @@ func resourceLogicalEnclosure() *schema.Resource {
 			"power_mode": {
 				Type:     schema.TypeString,
 				Optional: true,
+				Computed: true,
 			},
 			"scaling_state": {
 				Type:     schema.TypeString,
 				Optional: true,
```

Comment:
This is a read only field please remove optional from here.

Reply:
done